src/components/choose-locale/choose-locale.component.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { render, fireEvent, wait } from "@testing-library/react";
+import { ChangeLocale } from "./change-locale.component";
+import * as backendController from "./change-locale.resource";
+
+describe(`<ChangeLocale />`, () => {
+  const setupLocaleSelect = async () => {
+    const wrapper = render(
```

Comment:
Please read https://kentcdodds.com/blog/common-mistakes-with-react-testing-library .

In this particular case, the advice is: you don't need the return value from `render`; just use `screen`.